manual/manual/tutorials/objectexamples.etex

```diff
@@ -2,15 +2,20 @@
 \label{c:objectexamples}
 \pdfchapterfold{-15}{Tutorial: Objects in OCaml}
 %HEVEA\cutname{objectexamples.html}
-{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
+{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
```

Comment:
Am I a bit worried with this line, but this could be a problem with Github web interface. Could you check that this line does not exists in your local commit?

Reply:
It doesn't appear as a change when I diff the commit.  It may be that Github is expecting UTF-8 encoding for the accented characters (and would have multibyte representation) but that are probably in ISO 8851-1 (single byte).

![image](https://user-images.githubusercontent.com/1253341/35199791-1779a2fe-feb8-11e7-9085-7943b271d106.png)

Other changes applied as you suggested.